### PR TITLE
fix: links in example pipelines documentation

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
fix: links in example pipelines documentation

this PR fixes links in pipeline documentation (there was a refactoring of example pipelines in TTO which moved pipelines and configmaps to a different files).

**Release note**:
```
NONE
```

### CODE REVIEW

File: examples/pipelines/windows-bios-installer/README.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ This version is using templates, which are not available on Kubernetes.
...
 - [Windows 10 Installer PipelineRun for Kubernetes](windows10-installer-pipelinerun-kubernetes.yaml)
-- [Windows 10 Installer Pipeline for OKD](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows10-installer.yaml)
+- [Windows 10 Installer Pipeline for OKD](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows-bios-installer-pipeline.yaml)
 - [Windows 10 Installer PipelineRun for OKD](windows10-installer-pipelinerun-okd.yaml)
 
 ### Obtain Windows ISO download URL
 
 1. Go to https://www.microsoft.com/en-us/software-download/windows10ISO.
-   You can also obtain a server edition for evaluation at https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2019. (needs a different answer file!)
+   You can also obtain a server edition for evaluation at https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2019 (needs a different answer file!).
```
Comment (0xFelix) [inline examples/pipelines/windows-bios-installer/README.md:29]: Update to server 2k22?

Comment (ksimon1) [reply examples/pipelines/windows-bios-installer/README.md:29]: 2k22 can't use bios installer pipeline.

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Pull request merged**